develop/buildkite/pipeline.yml

```diff
@@ -15,9 +15,13 @@ steps:
     agents:
       queue: "default"
       docker: "*"
-    command: "make unit-test-coverage"
+    command: "make update-gotestsum unit-test-coverage"
+    env:
+      - GO_TEST: gotestsum
```

Comment:
are you going to do this for the functional tests too?

Reply:
That's the idea. I'm still in the middle of this.